src/containers/Home/Jobs/ListJob/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React,{useEffect,useState} from 'react'
+import Job from '../../../model/job';
+import { getJobs } from '../../../Redux/jobs/jobSlice';
+import {useSelector , useDispatch} from "react-redux"
+import { RootState } from '../../../Redux/store/store';
+export default function ListJob() {
+    const data = useSelector ((state:RootState) => state.jobs)
+    const dispatch = useDispatch();
+    useEffect(()=>{
+       dispatch(getJobs());
+    },[]);
+    const jobs= data.data;
+    const loading = data.loading;
+    if(loading){
```

Comment:
don't do like this.
Check the render Element into the return 
It looks like: 
`return (<>{loading ? <div>Loading</div> : <div>Some code here</div>}</>)`

Reply:
i've changed